services/brig/src/Brig/Team/API.hs

```diff
@@ -142,8 +143,23 @@ routesPublic = do
     Doc.response 200 "Invitation successful." Doc.end
     Doc.errorResponse invalidInvitationCode
 
+  head "/teams/invitations/by-email" (continue headInvitationPendingH) $
+    accept "application" "json"
+      .&. query "email"
+  document "HEAD" "headInvitationPending" $ do
+    Doc.summary "Check if there is an invitation pending given an email address."
+    Doc.parameter Doc.Query "email" Doc.bytes' $
+      Doc.description "Email address"
+    Doc.response 200 "Pending invitation exists." Doc.end
+    Doc.response 404 "No pending invitation exists." Doc.end
+
 routesInternal :: Routes a Handler ()
 routesInternal = do
+  -- TODO: Just needed for tests? Do we even need this?
```

Comment:
in tests it'd be ok to use the external end-point, no?

Reply:
yes